WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AuthenticatedWebView.swift

```diff
@@ -50,8 +50,7 @@ struct AuthenticatedWebView: UIViewRepresentable {
     }
 
     private func authenticatedPostData() throws -> URLRequest {
-        guard let username = credentials?.username,
-              let token = credentials?.authToken else {
+        guard case let .wpcom(username: username, authToken: token, siteAddress: _) = credentials else {
```

Comment:
Nit: this can be simplified too:

```suggestion
        guard case let .wpcom(username, token, _) = credentials else {
```

Reply:
Done as part of b231bcef7029ee63fab0c1e723c9e645ed2e16e1